contracts/ssnlist.scilla

```diff
@@ -673,14 +700,11 @@ procedure WithdrawalStakeAmt(initiator: ByStr20, ssn: ByStr20, withdraw_amount:
       HasBufferedDeposit ssn initiator;
       AdjustDeleg ssn initiator amt withdraw_amount;
       (* Illegal withdraw amount could be handled in procedure AdjustDeleg *)
-      DecreaseTotalStakeAmt withdraw_amount;
```

Comment:
Similarly, why change this code? It's not part of 6.1. Can we move it as a separate PR?